tsuru/client/apps.go

```diff
@@ -590,24 +629,53 @@ func ShortID(id string) string {
 	return id
 }
 
-func (a *app) String() string {
-	format := `{{ if .Error -}}
+const simplifiedFormat = `{{ if .Error -}}
 Error: {{ .Error }}
 {{ end -}}
 Application: {{.Name}}
-Description:{{if .Description}} {{.Description}}{{end}}
-Tags:{{if .TagList}} {{.TagList}}{{end}}
+{{- if .Description }}
+Description: {{.Description}}
+{{- end }}
+{{- if .TagList }}
+Tags: {{.TagList}}
+Created by: {{.Owner}}
+{{- end }}
+Platform: {{.Platform}}
+Plan: {{ .Plan.Name }}
+Pool: {{.Pool}} ({{ .Provisioner }}{{ if .Cluster}} cluster: {{ .Cluster }}{{end}})
+{{if not .Routers -}}
+Router:{{if .Router}} {{.Router}}{{if .RouterOpts}} ({{.GetRouterOpts}}){{end}}{{end}}
+{{end -}}
+Teams: {{.TeamList}}
+{{- if .InternalAddr }}
```

Comment:
I thought `.InternalAddr` always exists, but `.ExternalAddr` may exist or not.
If that's true, there is a missing `{{- if .ExternalAddr }}`